code/modules/economy/account.dm

```diff
@@ -15,6 +16,7 @@
 	var/welfare = FALSE
 	var/being_dumped = FALSE //pink levels are rising
 	var/withdrawDelay = 0
+	var/department_locked = FALSE //TRUE allows changing `account_department` into something else. used for VIP, Captain, and HoS
```

Comment:
Comment is wrong, true block's you from changing it

Reply:
fixed